contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol

```diff
@@ -443,8 +375,8 @@ contract OVM_StateCommitmentChain is iOVM_StateCommitmentChain, iRingBufferOverw
             "Invalid batch header."
         );
 
-        batches.deleteElementsAfterInclusive(
-            uint40(_batchHeader.batchIndex),
+        batches().del(
```

Comment:
Kept the name small here, but happy to go back to `deleteElementsAfterInclusive`. No preference.

Reply:
ah, beat me to it!